recipe/recipe-scripts-license.txt

```diff
@@ -0,0 +1,27 @@
+BSD-3-Clause license
```

Comment:
Interesting was this meant to be added?

Reply:
conda-smithy added it, and I don't oppose it, so yes.

Reply:
But I guess it's redundant.